test/functional/Java20andUp/src/org/openj9/test/jep434/downcall/MultiThreadingTests2.java

```diff
@@ -0,0 +1,130 @@
+	private static final FunctionDescriptor fd = FunctionDescriptor.of(structLayout, structLayout, structLayout);
+	private static final MemorySegment functionSymbol = SymbolLookup.loaderLookup().find("add2IntStructs_returnStruct").get();
+	private static final MethodHandle mh = Linker.nativeLinker().downcallHandle(functionSymbol, fd);
+
+	@Test(enabled=false)
+	@Override
+	public void uncaughtException(Thread thr, Throwable t) {
+		initException =  t;
+	}
+
+	@Test
+	public void test_twoThreadsWithSameFuncDesc_SharedDowncallHandler() throws Throwable {
+		Thread thr1 = new Thread(){
+			public void run() {
+				try {
+					VarHandle intHandle1 = structLayout.varHandle(PathElement.groupElement("elem1"));
+					VarHandle intHandle2 = structLayout.varHandle(PathElement.groupElement("elem2"));
+
+					try (Arena arena = Arena.openConfined()) {
+						SegmentAllocator allocator = SegmentAllocator.nativeAllocator(arena.scope());
+						MemorySegment structSegmt1 = allocator.allocate(structLayout);
+						intHandle1.set(structSegmt1, 11223344);
+						intHandle2.set(structSegmt1, 55667788);
+						MemorySegment structSegmt2 = allocator.allocate(structLayout);
+						intHandle1.set(structSegmt2, 99001122);
+						intHandle2.set(structSegmt2, 33445566);
+
+						MemorySegment resultSegmt = (MemorySegment)mh.invokeExact(allocator, structSegmt1, structSegmt2);
+						Assert.assertEquals(intHandle1.get(resultSegmt), 110224466);
+						Assert.assertEquals(intHandle2.get(resultSegmt), 89113354);
+					}
+				} catch (Throwable t) {
+					throw new RuntimeException(t);
+				}
+			}
+		};
+
+		Thread thr2 = new Thread(){
+			public void run() {
+				try {
+					VarHandle intHandle1 = structLayout.varHandle(PathElement.groupElement("elem1"));
+					VarHandle intHandle2 = structLayout.varHandle(PathElement.groupElement("elem2"));
+
+					try (Arena arena = Arena.openConfined()) {
+						SegmentAllocator allocator = SegmentAllocator.nativeAllocator(arena.scope());
+						MemorySegment structSegmt1 = allocator.allocate(structLayout);
+						intHandle1.set(structSegmt1, 11223344);
+						intHandle2.set(structSegmt1, 55667788);
+						MemorySegment structSegmt2 = allocator.allocate(structLayout);
+						intHandle1.set(structSegmt2, 99001123);
+						intHandle2.set(structSegmt2, 33445567);
+
+						MemorySegment resultSegmt = (MemorySegment)mh.invokeExact(allocator, structSegmt1, structSegmt2);
+						Assert.assertEquals(intHandle1.get(resultSegmt), 110224467);
+						Assert.assertEquals(intHandle2.get(resultSegmt), 89113355);
+					}
+				} catch (Throwable t) {
+					throw new RuntimeException(t);
+				}
+			}
+		};
+
+		thr1.setUncaughtExceptionHandler(this);
+		thr2.setUncaughtExceptionHandler(this);
+
+		thr1.start();
+		thr2.start();
+
+		thr1.join();
+		thr2.join();
+
+		if (initException != null){
+			throw new RuntimeException(initException);
+		}
```

Comment:
Missing space before `{` here and on lines 67 and 92.

Reply:
Fixed everywhere.